### DISCUSSION TITLE
vformat 错误

### DISCUSSION THREAD

**Discussion by zt449569708** [xmake-io/xmake#2586] [Q&A]
### Xmake 版本

2.6.9

### 操作系统版本和架构

ubuntu 18.04

### 描述问题

在xmake.lua中使用vformat，出现错误，xmake.lua和错误如下：
![Image_20220720171819](https://user-images.githubusercontent.com/6011216/179946100-735b989b-2356-4c59-9cc7-daf4da698f68.png)

### 期待的结果

vformat正确使用方式

**Comment by waruqi** [comment]
那是脚本域用的，描述域不要用，print 也不要在描述域用

**Comment by waruqi** [comment]
先看下相关文档 https://xmake.io/#/zh-cn/guide/syntax_description
